spec/services/fabricv4/patches/20240318_remove_additional_properties_from_SPAccessPointTypeVD.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/spec/services/fabricv4/oas3.patched/swagger.yaml b/spec/services/fabricv4/oas3.patched/swagger.yaml
+index 761c5fa..1c138bf 100644
+--- a/spec/services/fabricv4/oas3.patched/swagger.yaml
++++ b/spec/services/fabricv4/oas3.patched/swagger.yaml
+@@ -8238,6 +8238,7 @@ components:
+       required:
+         - type
+       type: object
++      additionalProperties: false
```

Comment:
Disabling additional properties is a bit risky; is this used for differentiation (i.e., there's some other schema that has the same properties as this one, plus some others)?

If the API ever adds a new property to this schema, that will break all previous releases of the SDK and customers will be forced to upgrade once an updated SDK is available.  That's arguably more of an API issue (rolling out breaking changes without a increasing the major version number), but something to keep in mind as these patches make their way to the official spec.

Reply:
Yes, it's for differentiation. The model has a counterpart `ServiceProfileAccessPointTypeCOLO` that has all the same base properties and more. The unmarshalling method fails to differentiate them when parsing into an `anyOf` that is using them both.

Reply:
Type differentiation is a difficult thing to spec in OpenAPI without introducing brittleness.  For example in the Metal spec we have places where we use `enum` to differentiate schemas:

```yaml
A:
  properties:
    type:
      type: string
      enum:
      - A
B:
  properties:
    type:
      type: string
      enum:
      - B
```

While that maintains the flexibility to add new properties to existing types, it still can cause problems for old clients if a new type is introduced.  I get the sense that the OpenAPI community generally avoids these kinds of constructs, and instead would create a schema that just has all of the properties for `A` and `B` and leaves it to the client to figure out what's what, but that has its own implications for customer experience.  If you come up with an approach for this for the Fabric spec, let me know!